contrib/cirrus/setup.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+UBUNTU_PACKAGES="
+    bzip2
+    cmake
+    coreutils
+    curl
+    gcc
+    git
+    openssl
+    pkg-config
+    python3
+    rsync
+    scons
+    vim
+    wget
+    yum-utils
+    zlib1g-dev
+    golang
+    go-md2man
+    libgpgme11-dev
+    libdevmapper-dev
+    libseccomp-dev
+    libglib2.0-dev
+    libostree-dev
+    bats
+    aufs-tools
+"
```

Comment:
Think we need btrfs-tools libpapparmor-dev libselinux-dev skopeo-containers (Maybe containers-common now) .
Would be nice to have these alphabetized as you have the ones in Fedora

Reply:
Added, and sorted.

Reply:
Actually adding those failed.  I'm going to leave sorting out the packaging as a TODO for when the testing is being implemented.  Added a comment to that effect.